### DISCUSSION TITLE
azure-devops-build-and-test-reports-using-odata-and-rest-in-powerbi/

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#21] [Giscussions]
# azure-devops-build-and-test-reports-using-odata-and-rest-in-powerbi/

All things DevOps and GitHub. Musings about DevOps tooling, culture and philosophy.

https://colinsalmcorner.com/azure-devops-build-and-test-reports-using-odata-and-rest-in-powerbi/

**Comment by BennyBrowns** [comment]
Really great article! Thanks for sharing :)

**Comment by prabhatnath** [comment]
Hi Colin, very good article. I am very new to power BI and working on an assignment to pull ADO Work Items data into PBI for a report. I am using ADO Analytics View but in that I am not able to get: Sprint Capacity, Acceptance Criteria and Description. I think these need API Call. Need your help/guidance on this. Please help.
